Extensions/mass_deleter.js

```diff
@@ -380,8 +383,11 @@ XKit.extensions.mass_deleter = new Object({
 
 		GM_xmlhttpRequest({
 			method: "GET",
-			url: "http://www.tumblr.com/likes/page/" + XKit.extensions.mass_deleter.unlike_likes_page + "?t=" + XKit.tools.random_string(),
```

Comment:
we should remove the unlike_likes_page variable now that it's unused.

Reply:
It's still used for the progress window.